src/Components/ListenOn/ListenOn.css

```diff
@@ -0,0 +1,28 @@
+.listenOnContainer {
+    align-items: center;
+    display: flex;
+    max-width: 1140px;
+    margin: auto;
+}
+
+h3 {
```

Comment:
since this will affect h3 as a global thing you may want to either give this h3 a specific class or style all h3 this way by moving this particular stlying to a more generic css file like `index.css`